recipes/mplcairo/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "mplcairo" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/mplcairo-{{ version }}.tar.gz
+  sha256: eac1d408cf4101db0ab4d0d57e6c3840bd5434818dc6cd295fc923062f9180d5
+
+build:
+  skip: true  # [win]
```

Comment:
Skipping Windows for now because I didn't have any luck getting it to build.  I'll revisit this later if I have time and create and issue about it on the feedstock in the meantime.